subsys/bluetooth/controller/ll_sw/nordic/lll/lll_conn.c

```diff
@@ -923,10 +934,6 @@ static void empty_tx_init(void)
 
 	p = (void *)radio_pkt_empty_get();
 	p->ll_id = PDU_DATA_LLID_DATA_CONTINUE;
-	p->cp = false;
```

Comment:
Please leave a comment here that other fields default values are zeros, so they are initialized automatically.